samples/Samples.h

```diff
@@ -168,6 +175,16 @@ typedef struct {
 
 typedef VOID (*StreamSessionShutdownCallback)(UINT64, PSampleStreamingSession);
 
+typedef struct {
+    MUTEX updateLock;
+    UINT64 lastAdjustmentTimeMs;
+    UINT64 currentVideoBitrate;
```

Comment:
This is in Kbps? Can we add Kbps to name to avoid any confusion?

Reply:
Not setting units because the unit is gstreamer encoder dependent. In most cases, video is in kbps and audio in bps, but do not want to assume. The actual value is read from with the gstreamer apis and that is what is modified. Since this is in the sample, customers are free to change these.